moodle/Sniffs/PHP/ForbiddenFunctionsSniff.php

```diff
@@ -65,5 +63,6 @@ class ForbiddenFunctionsSniff extends GenericForbiddenFunctionsSniff {
         'print_object' => null,
         // Dangerous functions. From coding style.
         'extract'      => null,
+        'unserialize'  => null,
```

Comment:
While I agree that we should avoid serialize and unserialize entirely, it might be nice to let people know that unserialize_object() is the Moodle-provided alternative to unserialize.
```suggestion
        'unserialize'  => 'unserialize_object',
```